src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('.tree');
+const items = list.querySelectorAll('li');
+
+items.forEach(element => {
+  const span = document.createElement('span');
+
+  element.prepend(span);
+  span.append(span.nextSibling);
+});
+
+list.addEventListener('click', e => {
+  const triger = e.target;
+  const listSibling = triger.nextElementSibling;
+
+  if (e.target.tagName === 'SPAN' && listSibling !== null) {
```

Comment:
Use 'trigger' instead of 'e.target' to be consistent with the variable defined above.

Reply:
actually not fixed, but not critical, so approved